modin/pandas/test/test_io.py

```diff
@@ -201,7 +201,8 @@ def eval_to_file(modin_obj, pandas_obj, fn, extension, **fn_kwargs):
                 last_exception = err
                 continue
             break
-        else:
+        # If we do have an exception that's valid let's raise it
+        if last_exception:
```

Comment:
this is kinda pointless, that `else:` would happen only when there was no `break` called within the loop, and _that_ can only happen if `continue` was called (3 times according to `range(3)`), and that in turn means there was at least one exception caught (actually, it was caught three times, but who cares)